promptsource/templates/piqa/templates.yaml

```diff
@@ -107,8 +172,9 @@ templates:
       _do_eval: false
       _do_train: true
       choices_in_prompt: true
-      metrics: []
-      original_task: null
+      metrics:
+      - Accuracy
+      original_task: true
     name: choose the most appropriate solution
```

Comment:
change that template so that Y is `answer_choices[label]` (i.e. outputting Solution 1 and 2)